contracts/access/AccessControl.sol

```diff
@@ -127,6 +127,7 @@ contract AccessControl {
      * - the caller must have ``role``'s admin role.
      */
     function setRoleAdmin(bytes4 role, bytes4 adminRole) external virtual admin(role) {
+        require(role != LOCK, "Use `lockRole`");
```

Comment:
Test, please. We always include a test for each `require` (or we should, I know I've cut corners before).

Reply:
Woah... After testing, I have strong reasons to believe this is not actually an issue, I will comment below.